client.go

```diff
@@ -8,6 +8,7 @@ import (
 	"errors"
 	"fmt"
 	"github.com/storageos/go-api/netutil"
+	"github.com/storageos/go-api/soserror"
```

Comment:
Can you change the package name not to uses the shortcut 'sos'

Reply:
I have renamed the package `serrors`, seems redundant to alias it every time.